src/Appium.Net/Appium/AppiumDriver.cs

```diff
@@ -190,13 +190,27 @@ public void PushFile(string pathOnDevice, FileInfo file) =>
         public void FingerPrint(int fingerprintId) =>
             AppiumCommandExecutionHelper.FingerPrint(this, fingerprintId);
 
-        public void BackgroundApp() =>
-            Execute(AppiumDriverCommand.BackgroundApp,
-                AppiumCommandExecutionHelper.PrepareArgument("seconds", -1));
+        public void BackgroundApp() {
+            Execute(DriverCommand.ExecuteScript, new Dictionary<string, object> {
+                ["script"] = "mobile:backgroundApp",
+                ["args"] = new object[] {
+                    new Dictionary<string, object> {
+                        ["seconds"] = -1
+                    }
+                }
+            });
+        }
 
-        public void BackgroundApp(TimeSpan timeSpan) =>
-            Execute(AppiumDriverCommand.BackgroundApp,
-                AppiumCommandExecutionHelper.PrepareArgument("seconds", timeSpan.TotalSeconds));
+        public void BackgroundApp(TimeSpan timeSpan) {
+                        Execute(DriverCommand.ExecuteScript, new Dictionary<string, object> {
+                ["script"] = "mobile:backgroundApp",
+                ["args"] = new object[] {
+                    new Dictionary<string, object> {
+                        ["seconds"] = timeSpan.TotalSeconds
+                    }
+                }
+            });
+        }
```

Comment:
You can replace these two overloads with a single method that takes a nullable TimeSpan? parameter. This will simplify things.